src/app/containers/MediaPlayer/index.jsx

```diff
@@ -128,12 +128,16 @@ const MediaPlayerContainer = ({
   const translatedNoJSMessage =
     path(['media', 'noJs'], translations) || noJsMessage;
 
+  const translatedExpiredContentMessage =
+    path(['media', 'contentExpired'], translations) ||
+    contentNotAvailableMessage;
```

Comment:
If we have the english message in service configs then do we need this fallback logic?

Reply:
I don't think so